lab-edward-smith/readme.MD

```diff
@@ -0,0 +1,64 @@
+# Cowsay As A Service
```

Comment:
Thanks for including a readme! Please be sure to include instructions for how to test your app in the future, thanks!